typedoc.js

```diff
@@ -1,25 +1,8 @@
 module.exports = {
-  exclude: ['**/*spec.ts', './src/index.ts', './src/wasm/wasm_exec.js'],
+  exclude: ['**/testSetup/*', '**/*.spec.ts', './src/wasm/wasm_exec.js'],
   listInvalidSymbolLinks: true,
-  stripInternal: true,
+  excludeExternals: true,
   excludeNotExported: true,
   excludePrivate: true,
-  hideGenerator: true,
-  mode: 'file',
-  tsconfig: 'tsconfig.json',
-  readme: 'README.md',
```

Comment:
Why remove the Readme?

Reply:
I thought it was redundant. The readme is already in the github project. We can add it back if you want.